src/apic-extension/azext_apic_extension/tests/latest/recordings/test_create_service.yaml

```diff
@@ -0,0 +1,102 @@
+interactions:
```

Comment:
Are tests live in pipeline or just new recordings?

Reply:
My understanding is CLI requires us to provide recording when adding test cases, so CI can run in playback mode. And there's already recording files in this folder. Please let me know if you think we shouldn't add recordings.